tests/unit/command/test_machine.py

```diff
@@ -119,11 +120,37 @@ def test_modified(tmp_dir):
     assert config['machine "foo"']["cloud"] == "azure"
 
 
-def test_rename(tmp_dir, scm, dvc):
+def test_eename(tmp_dir, scm, dvc):
```

Comment:
```suggestion
def test_rename(tmp_dir, scm, dvc):
```

Looks like a typo got committed here?